### PR TITLE
1st pass at site doc changes for ec es go live

### PR DESCRIPTION
**Pull request by ChrisMcGowan**
## Changes proposed in this pull request:
- Updated aws ec/es docs for prod ready
- Updated legacy redis and es docs to prep for depreciation
-

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/go_live_eces)

## Security Considerations
n/a

### CODE REVIEW

File: _docs/services/elasticsearch56.md [hunk 1]
```diff
@@ -5,16 +5,18 @@ sidenav: true
 title: "Elasticsearch"
 name: "elasticsearch56"
 description: "Elasticsearch version 5.6: a distributed, RESTful search and analytics engine"
-status: "Beta"
 ---
 
+Note - this service is being deprecated in favor of the [aws-elascticsearch]({{ site.baseurl }}{% link _docs/services/eaws-elasticsearch.md %}).
```
Comment (bengerman13) [inline _docs/services/elasticsearch56.md]: this link is broken - `eaws` -> `aws`

File: _docs/services/redis.md [hunk 2]
```diff
@@ -9,13 +9,15 @@ redirect_from:
 - /docs/services/redis32
 ---
 
+Note - this service is being deprecated in favor of the [aws-elascticahe]({{ site.baseurl }}{% link _docs/services/eaws-elasticache.md %}).
```
Comment (bengerman13) [inline _docs/services/redis.md]: `eaws` -> `aws`

File: _docs/services/elasticsearch56.md (hunk 1 above)
Comment (ChrisMcGowan) [reply _docs/services/elasticsearch56.md]: thanks @bengerman13 fixed in new commit

File: _docs/services/redis.md (hunk 2 above)
Comment (ChrisMcGowan) [reply _docs/services/redis.md]: thanks @bengerman13 fixed in new commit

**Comment by ChrisMcGowan** [comment]
@RonWilliams let me know if you would like any more changes before go live

**Review by RonWilliams — APPROVED** [review]

**Pull request merged**